tests/Browser/Actions/Test.php

```diff
@@ -119,4 +119,34 @@ public function test()
             ;
         });
     }
+
+    public function test_button_dom_update_click_event()
+    {
+        $this->browse(function ($browser) {
+            Livewire::visit($browser, Component::class)
+                /**
+                 * sibling buttons
+                 */
+                ->press('@show.button.actions')
+                ->waitForLivewire()->pause(100)->assertPresent('@button.with-actions')
+                ->press('@button.with-click')
+                ->waitForLivewire()->pause(100)->assertSeeIn('@output', 'button with wire:clicked got triggered')
```

Comment:
thats not how `waitForLivewire()` is designed
For multiple assertions and actions before livewire renders AGAIN
```php
waitForLivewire(function ($b) {
    $b->click('something')
})
// livewire has done its magic and you proceed here after all is rendered
```

sometimes its just a oneliner so you can just pass the single action like HigherOrderProxy stuff
```php
waitForLivewire()->click('something')
// livewire has done its magic and you proceed here after all is rendered
```

```php
            Livewire::visit($browser, Component::class)
                /**
                 * sibling buttons
                 */
                ->waitForLivewire()->press('@show.button.actions')
                ->assertPresent('@button.with-actions')
                ->waitForLivewire()->press('@button.with-click')
                ->assertSeeIn('@output', 'button with wire:clicked got triggered')
            ;
```

just to streamline, we use `click()` over `press()` in 99%